server/export.go

```diff
@@ -51,12 +50,9 @@ func ExportCmd(ctx *Context, cdc *codec.Codec, appExporter AppExporter) *cobra.C
 			if err != nil {
 				return err
 			}
-			height := viper.GetInt64(flagHeight)
-			if height < 0 {
-				return errors.Errorf("Height must greater than or equal to zero")
-			}
+
 			forZeroHeight := viper.GetBool(flagForZeroHeight)
-			appState, validators, err := appExporter(ctx, ctx.Logger, db, traceWriter, height, forZeroHeight)
+			appState, validators, err := appExporter(ctx, ctx.Logger, db, traceWriter, forZeroHeight)
```

Comment:
Why do you remove height from export option? It is possible to export state for zero height but not at current height.

Reply:
> Why do you remove height from export option? It is possible to export state for zero height but not at current height.
If you want to export state for zero or snapshoot height, reset it firstly.